src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      <p className="Person__age">{age ? (`I am ${age}`) : null}</p>
+      <p className="Person__partner">{isMarried ? (`${partnerName} is my ${sex === 'm' ? ('wife') : ('husband')}`) : (`I am not married`) }</p>
```

Comment:
This comment is not fixed
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/99403198/221347752-68fa0864-158c-45a2-a4e6-2abacfbf5fd8.png">

```
const partner = sex === 'm' ? 'wife' : 'husband';
....

<p className="Person__partner">
  {isMarried ? (`${partnerName} is my ${partner) : (`I am not married`) }
</p>